package.json

```diff
@@ -73,12 +73,13 @@
     "create-react-context": "^0.2.1",
     "deepmerge": "^2.1.1",
     "flow-copy-source": "^2.0.2",
-    "hoist-non-react-statics": "^2.5.0",
+    "hoist-non-react-statics": "2.5.0",
     "prop-types": "^15.6.0"
   },
   "jest": {
     "moduleNameMapper": {
       "\\.(css|less)$": "<rootDir>/__mocks__/styleMock.js"
-    }
+    },
+    "testURL": "http://localhost/"
```

Comment:
🤔

Reply:
without it tests don't work.

Reply:
https://github.com/jsdom/jsdom/issues/2304

Reply:
https://github.com/jsdom/jsdom/issues/2304